eng/common/scripts/Prepare-Release.ps1

```diff
@@ -177,8 +177,23 @@ if ($releaseTrackingOnly)
 
 if (Test-Path "Function:SetPackageVersion")
 {
-  SetPackageVersion -PackageName $packageProperties.Name -Version $newVersion -ServiceDirectory $packageProperties.ServiceDirectory -ReleaseDate $releaseDateString `
-    -PackageProperties $packageProperties
+  $latestVersion = Get-LatestReleaseDateFromChangeLog -ChangeLogLocation $packageProperties.ChangeLogPath
+  if ($latestVersion -ne $null)
+  {
+    $promptMessage = "The latest entry in the CHANGELOG.md already has a release date. Do you want to replace the latest entry title? Please enter (y or n)."
+    while (($readInput = Read-Host -Prompt $promptMessage) -notmatch '^[yn]$'){ }
+    $replaceLatestEntryTitle = ($readInput -eq "y")
+
+    SetPackageVersion -PackageName $packageProperties.Name -Version $newVersion `
+      -ServiceDirectory $packageProperties.ServiceDirectory -ReleaseDate $releaseDateString `
+      -PackageProperties $packageProperties -ReplaceLatestEntryTitle $replaceLatestEntryTitle
+  }
+  else
+  {
+    SetPackageVersion -PackageName $packageProperties.Name -Version $newVersion `
```

Comment:
Why do we need to duplicate this SetPackageVersion call? YOu should be able to just make the call once with the correct replaceLatestEntryTitle value.